olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -781,7 +781,7 @@ spec:
                 containers:
                   - name: knative-operator
                     # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
-                    image: registry.ci.openshift.org/knative/openshift-serverless-v1.29.0:openshift-knative-operator
+                    image: registry.ci.openshift.org/knative/openshift-serverless-nightly:openshift-knative-operator
```

Comment:
Hmm. Should this be `openshift-serverless-v1.29.1` instead of openshift-serverles-nightly now that https://github.com/openshift/release/pull/40466/files was merged? Same for a few places below in this PR.
Or you plan to create another PR after this one to fix it?

Reply:
Not sure why automated script not replace with current version. Not debug properly replace it manually.